app/pb_kits/playbook/pb_list/docs/_list_xpadding.html.erb

```diff
@@ -1,7 +1,7 @@
 <%= pb_rails("list", props: {ordered:false, dark:false, borderless:false, xpadding:true}) do %>
-  <%= pb_rails("list/item") %>
-  <%= pb_rails("list/item") %>
-  <%= pb_rails("list/item") %>
-  <%= pb_rails("list/item") %>
-  <%= pb_rails("list/item") %>
+<%= pb_rails("list/item") do %> List Item <% end %>
```

Comment:
Moving forward, depending on our use cases, it might be worth allowing for a shorthand:

```rb
<%= pb_rails("list", props: {
  ordered: true,
  dark: false,
  borderless: false,
  items: my_items,
  item_renderer: "list/item" # this prop could also have a sensible default value, perhaps "list/item"?
}) %>
```

Reply:
@kre8sions This is kind of what I was mentioning for simple lists where we just pass data. Nice @drborges